### DISCUSSION TITLE
venus v1.0.4 Release Timeline

### DISCUSSION THREAD

**Discussion by Joss-Hua** [filecoin-project/venus#4509] [Announcements]
This is an optional version of Venus v1.10, which introduces many improvements and fixes. Will remain updated, highlights are:

- github.com/filecoin-project/venus (v1.0.3 -> v1.0.4):
  * https://github.com/filecoin-project/venus/pull/4506

- github.com/filecoin-project/venus-sealer (v1.2.4 -> v1.2.5):
  *  https://github.com/filecoin-project/venus-sealer/pull/65
  * https://github.com/filecoin-project/venus-sealer/pull/66
  * https://github.com/filecoin-project/venus-sealer/pull/67
  * https://github.com/filecoin-project/venus-sealer/pull/68

- Fix FinalizeSector with sectors in storage paths 

- `AggregateAboveBaseFee` is added to miner sealing configuration for setting the network base fee to start aggregating proofs. When the network base fee is lower than this value, the prove commits will be submitted individually via `ProveCommitSector`

- Merge the necessary updates brought by some Lotus updates
__________

If you have any concerns or questions, leave a comment here.

### Tentative release timeline
> Time below is in UTC.

- [x] 2021-07-30, [venus v1.0.4](https://github.com/filecoin-project/venus/releases/tag/v1.0.4)
- [x] 2021-07-30, [venus-sealer v1.2.5](https://github.com/filecoin-project/venus-sealer/releases)

**Comment by diwufeiwen** [comment]
2021-08-05, venus-sealer v1.2.6,Fix the problems found in the recent access to the incubator.

**Comment by Joss-Hua** [comment]
[venus-seaelr v1.2.6](https://github.com/filecoin-project/venus-sealer/releases/tag/v1.2.6) has been released, which optimizes some problems feedback by participants in the incubation center, making the distributed storage pool of the incubation center further successful.

- sealer: skip 2k winningpost & verify  (https://github.com/filecoin-project/venus-sealer/pull/85)
- update README.md for cmd use  (https://github.com/filecoin-project/venus-sealer/pull/82)
- Fix the issue of recent incubation feedback  (https://github.com/filecoin-project/venus-sealer/pull/76)
- fix repository init error  (https://github.com/filecoin-project/venus-sealer/pull/80)
- reduce list sectors time  (https://github.com/filecoin-project/venus-sealer/pull/79)
- fix ticket expire check  (https://github.com/filecoin-project/venus-sealer/pull/78)
